### ISSUE TITLE
Include an image in the example front end

### ISSUE THREAD

**Issue by ubershmekel** [gothinkster/realworld#101]
I believe including an image to be displayed would be a bit more real-world. I realize every frontend and backend might do it differently, but that is kind of what you're going for here perhaps.

**Comment by SandeeshS** [comment]
@ubershmekel Can you be a bit more elaborate? Like including an image with the article or just having an image section?

**Comment by ubershmekel** [comment]
I think even just including a logo for the website would be beneficial. To demonstrate how images would be stored, and referenced in the frontend. E.g. for an interestingly different example:

http://stackoverflow.com/questions/32612912/dynamically-add-images-react-webpack

```
import LogoImg from 'YOUR_PATH/logo.png';

...

<img src={LogoImg}/>
```

**Comment by SandeeshS** [comment]
Ah got it. Guess they wanted to keep the design simple and skipped adding a logo. We could definitely add a logo in the next version of the spec along with the other changes planned.

**Comment by EricSimons** [comment]
Yup, we wanted to keep it simple for v1 but tbh, if we did this, we'd probably just link to a CDN resource instead of having the FE's handle asset resolution/bundling themselves (just like we did with the main.css stylesheet). Reason being is that the way ppl deploy assets varies greatly and we don't want to lock people into one way of handling things.

However, this seems like an excellent ["extra credit"](https://github.com/gothinkster/realworld/issues/90) objective that folks could find useful — so if someone out there wants to fork one of the base repo's and show how it's done, we'd love to link to it!
